commands/ls.go

```diff
@@ -232,11 +233,16 @@ func matchesName(host *host.Host, names []string) bool {
 func attemptGetHostState(h *host.Host, stateQueryChan chan<- HostListItem) {
 	url := ""
 	hostError := ""
+	dockerVersion := "?"
 
 	currentState, err := h.Driver.GetState()
 	if err == nil {
 		url, err = h.GetURL()
 	}
+	if err == nil {
+		dockerVersion, err = h.GetDockerVersion()
```

Comment:
This would print something like that:

```
NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   VERSION   ERRORS
default   -        virtualbox   Running   tcp://192.168.99.100:2376                     Unable to query docker version: Get https://192.168.99.100:2376/v1.15/version: dial tcp 192.168.99.100:2376: getsockopt: connection refused
```

So it doesn't hang. Also, we call this piece of code only when the host is running and has a url.